src/quo/components/profile/collectible_list_item/view.cljs

```diff
@@ -0,0 +1,208 @@
+  (let [theme (quo.theme/use-theme-value)]
+    [rn/view {:style (style/card-view-container theme)}
+     [rn/view
+      (cond
+        (= :loading status)
+        [loading-image
+         {:theme                theme
+          :gradient-color-index gradient-color-index
+          :height               style/type-card-image-height-and-border}]
+
+        (= status :unsupported)
+        [fallback
+         {:theme theme
+          :width width
+          :type  type
+          :label (i18n/label :t/unsupported-file)}]
+
+        (= status :cant-fetch)
+        [fallback
+         {:theme theme
+          :width width
+          :type  type
+          :label (i18n/label :t/cant-fetch-info)}]
+
+        :else [rn/image
+               {:style  (style/card-image width)
+                :source image-src}])]
+     (when (and (not= status :loading) (not= status :cant-fetch) counter)
+       [collectible-counter/view
+        {:container-style style/collectible-counter
+         :size            :size-24
+         :value           counter}])
+     [card-details
+      {:status           status
+       :community?       community?
+       :avatar-image-src avatar-image-src
+       :collectible-name collectible-name
+       :theme            theme}]]))
+
+(defn- image-view
+  [{:keys [avatar-image-src community? counter
+           gradient-color-index image-src status width]}]
+  (let [theme (quo.theme/use-theme-value)]
+    [rn/view {:style style/image-view-container}
+     (cond
+       (= :loading status)
+       [loading-image
+        {:theme                theme
+         :gradient-color-index gradient-color-index
+         :height               style/type-image-height}]
+
+       (= status :unsupported)
+       [fallback
+        {:theme theme
+         :width width
+         :type  type
+         :label (i18n/label :t/unsupported-file)}]
+
+       (= status :cant-fetch)
+       [fallback
+        {:theme theme
+         :width width
+         :type  type
+         :label (i18n/label :t/cant-fetch-info)}]
+
+       :else [rn/image
+              {:style  {:width         width
+                        :height        width
+                        :border-radius style/container-border-radius}
+               :source image-src}])
+     (when (and (not= status :loading) (not= status :cant-fetch) counter)
+       [collectible-counter/view
+        {:container-style style/collectible-counter
+         :size            :size-24
+         :value           counter}])
+     (when (and (not= status :loading) (not= status :cant-fetch) community?)
+       [preview-list/view
+        {:container-style style/avatar
+         :type            :communities
+         :size            :size-24}
+        [avatar-image-src]])]))
+
+"
+width is dynamic based on available space and has to be passed to the component.
+This will used in large lists, to avoid recalculating the same value it is easier to pass in the value. 
+"
```

Comment:
Question (maybe not the right thread to ask, or I misunderstood this `width` prop. Correct me if I'm wrong)

If it's just the padding on the sides (vertical) that needs to be consistent across different screens and the component is used only in the list (taking the width of available of space), I believe we can use the default flex along with the `:num-columns` prop in `FlatList`.

Reply:
gonna look into it again if I can use flex here. If I recall correctly it was mostly the "type" `:card` variant but perhaps I just fix the height on the bottom part (the text details and avatar image) and then leave the upper part be square.
But I think there was more than that, it was some of the empty content just wouldn't stretch for me, will have to recheck/revisit this.

Reply:
Yeah, I was exactly thinking about the `:num-columns` prop @smohamedjavid mentioned.

@J-Son89 the current collectible listing is using that prop, just if you want an example, but I'm not sure if it can be applied to this one.

> will have to recheck/revisit this.

Thanks for considering it @J-Son89 :+1:

Reply:
addressed this and now not using a fixed width. It's all responsive 👍